src/test/java/Homework19.java

```diff
@@ -0,0 +1,18 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest{
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        launchBrowser("https://bbb.testpro.io/");
```

Comment:
The launchBrowser() method is annotated as @BeforeMethod so you don't need to call it. It will be executed automatically before @Test method